models/library-imported-from-ecore.json

```diff
@@ -0,0 +1,254 @@
+[
```

Comment:
Is this also generated from the one in ecore?

Reply:
Yup. And it's checked that it's equal to the serialization of the hand-crafted version - that works because of deterministic ID generation.